python/dolfinx/fem/dofmap.py

```diff
@@ -19,24 +19,46 @@ def __init__(self, dofmap: cpp.fem.DofMap):
         self._cpp_object = dofmap
 
     def cell_dofs(self, cell_index: int):
+        """ Returns the Local-global dof map for the cell (using process-local indices)
+
+        Parameters
+        ----------
+        cell
+          The cell index
+        """
         return self._cpp_object.cell_dofs(cell_index)
 
     @property
     def bs(self):
+        """
```

Comment:
Can we put the one-line doc strings on one line? Three lines is a waste of space and reduced code readability.